src/data/data.js

```diff
@@ -13,7 +13,7 @@ const pageData = {
           'Promovemos la recolección del aceite usado de cocina, para que pueda ser utilizado nuevamente en una energía renovable.',
         button: {
           text: 'Registrate',
-          href: 'Register',
+          href: '/register.html',
```

Comment:
In the navigation links you have `/registro.html`, but here you have `/register.html`. These two links need to match. Please update.

Reply:
Done!!!